### PR TITLE
updates the IC filter

### PR DESCRIPTION
**Pull request by ErwinSnowden**
## About The Pull Request
adds new prohibited words to the IC filter, most of which are modern day internet terminology that i've seen used in game

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
add: bad terrible words that should never be uttered in game to the filter.
/:cl:

### CODE REVIEW

**Comment by ErwinSnowden** [comment]
no clue why this is failing integration tests

**Comment by Al-1ce** [comment]
Surely we might consider dealing with the people who say stupid shit like this constantly and have no interest in roleplay?

**Comment by ErwinSnowden** [comment]
> Surely we might consider dealing with the people who say stupid shit like this constantly and have no interest in roleplay?

i am doing what i am able to, but it seems to be helpless. if i code mobs to kill people, staff will just resurrect them.

the devs care for the game while the staff do not, so we must take code measures to stop people like "black bull" from chanting kill whitey.

File: config/in_character_filter.txt [hunk 1]
```diff
@@ -40,3 +42,15 @@ hrp
...
+clone
+sneed
+whitey
+wyci
+nft
+cryptocurrency
+redpill
+blackpill
+bluepill
+thybaron
```
Comment (BadAtThisGame302) [inline config/in_character_filter.txt:56]: wtf did thybaron do to yo dumbass bro???

Comment (BadAtThisGame302) [reply config/in_character_filter.txt:56]: can we add kimitus to the ic filter

Comment (ErwinSnowden) [reply config/in_character_filter.txt:56]: okay, im making a compromise, we are adding bob newman to the ic filter and that's final.

**Comment by ErwinSnowden** [comment]
> why "ladyboy"?

its a slur

File: config/in_character_filter.txt (hunk 1 above)
Comment (RonBag) [reply config/in_character_filter.txt:56]: no names of f13 community members in the chat filter........

**Comment by Cartlord** [comment]
Maybe I just roll in the wrong circles, but I've never heard it used that way - hell, I've barely heard it used EVER.

**Pull request closed without merging**